From 08376b2fbaecd6ae77e2c6fc97be6e838a81c91d Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Tue, 11 Aug 2026 20:59:01 +0200
Subject: [PATCH] dist: Fully lock the `tss` system user

Use the `u!` notation to fully lock the system user.
It is highly unlikely that the `tss` user should ever be logged into
(e.g. via SSH).

This sysusers.d feature has been released with systemd 257 on
2024-12-10.

Related-to: https://github.com/systemd/systemd/releases/tag/v257
Signed-off-by: David Runge <dave@sleepmap.de>
---
 dist/sysusers.d/tpm2-tss.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dist/sysusers.d/tpm2-tss.conf b/dist/sysusers.d/tpm2-tss.conf
index 3ef99a74..3670482d 100644
--- a/dist/sysusers.d/tpm2-tss.conf
+++ b/dist/sysusers.d/tpm2-tss.conf
@@ -1,2 +1,2 @@
 #Type Name ID GECOS               Home directory Shell
-u     tss  -  "tss user for tpm2"
+u!     tss  -  "tss user for tpm2"
